purchase_order_line_deep_sort/models/purchase_order.py

```diff
@@ -45,6 +46,8 @@ def resolve_subfields(obj, line_order):
         )
         for line in sorted_lines:
             sequence += 10
```

Comment:
Magic number...

Reply:
Yes, as it's a suitable gap between lines. This is used in Odoo in other places. Any better suggestion?

Reply:
It's recommended to use a constant

Reply:
Ah, well, I get you. OK, as this is out of this PR, let's merge it and we can improve it later with that constant for being in header.